src/main/java/warehouse/Cart.java

```diff
@@ -1,4 +1,4 @@
-package main.java.warehouse;
+package warehouse;
```

Comment:
Se cambi i package in questo modo, penso che a tutti noi smetta di funzionare il codice

Reply:
allora, come ha detto lorenzo ho pushato nel branch dove corretto il metodo sulla media semplicemente le modifiche a quel metodo. Se approvate ci togliamo questa PR così apro un altro branch per risolvere il problema dei package